### ISSUE TITLE
[PHP7] php7.0-dev installs PHP5

### ISSUE THREAD

**Issue by HanXHX** [gplessis/dotdeb-php#119]
Hi @gplessis,

When I install `php7.0-dev`, it installs php5.

```
vagrant@debiancontrib-jessie:~$ apt-cache policy php7.0-dev
php7.0-dev:
  Installed: (none)
  Candidate: 7.0.3-1~dotdeb+8.1
  Version table:
     7.0.3-1~dotdeb+8.1 0
        500 http://packages.dotdeb.org/ jessie/all amd64 Packages
     7.0.2-1~dotdeb+8.2 0
        500 http://packages.dotdeb.org/ jessie/all amd64 Packages
     7.0.2-1~dotdeb+8.1 0
        500 http://packages.dotdeb.org/ jessie/all amd64 Packages
     7.0.1-1~dotdeb+8.1 0
        500 http://packages.dotdeb.org/ jessie/all amd64 Packages
     7.0.0-5~dotdeb+8.1 0
        500 http://packages.dotdeb.org/ jessie/all amd64 Packages
     7.0.0-1~dotdeb+8.1 0
        500 http://packages.dotdeb.org/ jessie/all amd64 Packages
```

```
vagrant@debiancontrib-jessie:~$ apt-cache depends php7.0-dev
php7.0-dev
  Depends: autoconf
  Depends: automake
  Depends: libpcre3-dev
  Depends: libssl-dev
  Depends: libtool
  Depends: php7.0-cli
  Depends: php7.0-common
  Depends: shtool
  Recommends: <dh-php7.0>
  Recommends: pkg-php-tools
  Conflicts: php5-dev
```

```
vagrant@debiancontrib-jessie:~$ sudo apt-get install php7.0-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  libonig2 libpcre3-dev libpcrecpp0 libssl-dev libssl-doc php-pear php5-cli php5-common php5-json php5-readline pkg-php-tools shtool zlib1g-dev
Suggested packages:
  php5-dev php5-user-cache dh-make
Recommended packages:
  dh-php7.0
The following NEW packages will be installed:
  libonig2 libpcre3-dev libpcrecpp0 libssl-dev libssl-doc php-pear php5-cli php5-common php5-json php5-readline php7.0-dev pkg-php-tools shtool zlib1g-dev
0 upgraded, 14 newly installed, 0 to remove and 16 not upgraded.
Need to get 0 B/7,106 kB of archives.
After this operation, 27.5 MB of additional disk space will be used.
```

The problem comes from `php7.0-dev`. It recommends `pkg-php-tools` which depends php5... I think you should remove pkg-php-tools from recommends.

```
vagrant@debiancontrib-jessie:~$ apt-cache depends pkg-php-tools
pkg-php-tools
  Depends: debhelper
  Depends: php-pear
  Depends: php5-cli
  Depends: php5-json
  Suggests: dh-make
```

```
vagrant@debiancontrib-jessie:~$ apt-cache policy pkg-php-tools
pkg-php-tools:
  Installed: (none)
  Candidate: 1.28
  Version table:
     1.28 0
        500 http://httpredir.debian.org/debian/ jessie/main amd64 Packages
```

Cheers!

Emilien

**Comment by HanXHX** [comment]
My sources.list:

```
vagrant@debiancontrib-jessie:~$ grep '^[^#$]' /etc/apt/sources.list
deb http://httpredir.debian.org/debian jessie main
deb-src http://httpredir.debian.org/debian jessie main
deb http://security.debian.org/ jessie/updates main
deb-src http://security.debian.org/ jessie/updates main
vagrant@debiancontrib-jessie:~$ grep '^[^#$]' /etc/apt/sources.list.d/*
deb http://packages.dotdeb.org jessie all
deb-src http://packages.dotdeb.org jessie all
```

You can reproduce the problem on my project (php7 branch): https://github.com/HanXHX/ansible-php

You need ansible, vagrant and virtualbox. For check, just "vagrant up debian jessie-php-7.0"

**Comment by gplessis** [comment]
I need to investigate the implications of such a move. In the meantime, you can easily circumvent this issue by using `apt-get install --no-install-recommends php7.0-dev`

**Comment by gplessis** [comment]
This will be fixed with the upcoming PHP 7.0.10 packages. Thank for your feedback.
